### ISSUE TITLE
NodeJS doesn't exit after retrieved list

### ISSUE THREAD

**Issue by vithd** [facundoolano/google-play-scraper#137]
Hi, I'm having problem after getting list. My app doesn't stop after the list is shown.

```
var gplay = require('google-play-scraper');

gplay.list({
    category: gplay.category.GAME_ACTION,
    collection: gplay.collection.TOP_FREE,
    num: 2
  })
  .then(console.log, console.log);
```

**Comment by facundoolano** [comment]
this has something to do with memoization (doens't happen with `cache: false`). Looking into it.

**Comment by facundoolano** [comment]
I'm thinking this may be an expected or known issue when using memoization. If the script is short lived, it shouldn't do memoization. This issue should be covered by #142

**Comment by andreilima** [comment]
I solved it by adding 
`.then(result => {
  console.log(result, '\n');
  process.exit(1);
}, console.log);
`

**Comment by icaroponce** [comment]
The problem seems to occur always that the maxAge is set in the config object to memoizee function. The application lasts the same time to exit as it's set on this maxAge argument. if I set maxAge to 5000, the script won't exit until this time is finished ( 5s + the real time that the operation should have took ). 

Using the v3.2.4, my memoization is still be used by default, I have to set cache: false to every method i call.

**Comment by Trane9991** [comment]
Example
```
var gplay = require('google-play-scraper');

gplay.app({appId: 'com.dxco.pandavszombies'})
  .then(console.log, console.log);
``` 
doesen't return too.

**Comment by facundoolano** [comment]
I removed default memoization, so this should be fixed in the latest version (4.0.0)
